server/controllers/errors/500.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-unused-vars */
+import { ErrorRequestHandler } from 'express';
+
+const serverError: ErrorRequestHandler = (error, req, res, next) => {
+  res.status(error.status).json({ msg: error.msg });
```

Comment:
The error you receive here could be an error thrown anywhere from your server, so it could not have any status code or error message, so you need to check first